docs/cookbook.rst

```diff
@@ -249,3 +249,33 @@ Unless of course something bad happens inside of ``PerformA`` in which case the
     AssertionError:
     Expected: <True>
          but: was <False>
+
+
+Using Either
+===========
+
+Sometimes we need to try one thing then followed by a different thing when the first one fails.
+You could just add a try/except in your code but that breaks up the screenplay pattern.
+So there is an action for that; :func:`~screenpy.actions.Either`::
+
+    the_actor.will(Either(DoAction()).or_(DoDifferentAction())
+
+Screenpy will attempt to perform the first action but if an `AssertionError` is raised
+screenpy will move on to attempt performing the second action instead.  Note that we only catch
+`AssertionError` here allowing for other exceptions to still be raised.
+
+The action also allows users to pass in multiple actions similar to how actors can perform
+multiple actions in one call::
+
+    the_actor.will(
+        Either(
+            DoAction1(),
+            DoAction2(),
+            DoAction3(),
+        ).or_(
+            DoDifferentAction1(),
+            DoDifferentAction2(),
+            DoDifferentAction3(),
+        )
+    )
```

Comment:
This is where I think decent documentation should illustrate the limitations of the action.  It should be understood by the tester that it will execute all of the actions and should any of them raise AssertionError, it will execute the 2nd set.  We might want to give a more complex scenario and point out perhaps `Either` isn't the right fit.  
 
 OR... maybe make the example less abstract?  It get's a little harder to give real-word examples without situational context and evoking the other packages but something like this?
 
 ```python
the_actor.will(
    Either(
        See(BrowserURL(), HasUrlNetloc(URL)),
        CheckIfAuthenticated(),
    ).or_(
        ClearCache()
        Open.their_browser_on(URL())
        Eventually(
            Enter(username).into(USERNAME_FIELD),
            Enter.the_secret(password).into(PASSWORD_FIELD),
            Click.on(SIGN_IN_BUTTON)
        )
    )
)
```

Reply:
Maybe we can call it out as a note, i think you can do those fancy popout things in RST...
```rst
.. note::
   :class:`~screenpy.actions.Either` will not describe any cleanup for the Actor
   after it experiences a failure in the first routine; the Actor will proceed directly
   to the second routine. Keep this in mind while defining the two branches of Actions.
```

Reply:
Either way (pun intended), i do agree that we should have a more concrete example here. You could use Actions from the example tutorial, with `cam_py` and `pollster`.

Reply:
The latest version has the above example added.